paper/main.tex

```diff
@@ -1471,6 +1449,16 @@ \subsubsection*{Reverse inference}
 associated with the average dynamic correlation patterns at each
 order.
 
+\section*{Data Availability}
+
+The authors declare that the data supporting the findings of this
+study as well as the source data for this paper are available at \href{https://github.com/ContextLab/timecorr-paper/releases/tag/v0.3}{github.com/ContextLab/timecorr-paper/releases/tag/v0.3}. The fMRI dataset collected by \cite{SimoEtal16} has been made publicly available at
+\href{http://arks.princeton.edu/ark:/88435/dsp015d86p269k}{arks.princeton.edu/ark:/88435/dsp015d86p269k}
+
+\section*{Code Availability}
+
+All of our analysis code may be downloaded from\href{https://github.com/ContextLab/timecorr-paper/releases/tag/v0.3}{github.com/ContextLab/timecorr-paper/releases/tag/v0.3}.
+
```

Comment:
Missing space after "from"

Also should we cite timecorr toolbox?  E.g. you could add something like:

"We have also published a companion Python toolbox that may be downloaded from \href{https://timecorr.readthedocs.io}{timecorr.readthedocs.io}."

Reply:
spaced

Reply:
done!  sorry didn't see that last part earlier